resources/definitions/flashforge_dreamer_nx.def.json

```diff
@@ -0,0 +1,48 @@
+{
+    "version": 2,
+    "name": "Dreamer NX",
+    "inherits": "flashforge_base",
+    "metadata": {
+        "visible": true,
+        "author": "Egon",
+        "manufacturer": "Flashforge",
+        "file_formats": "text/x-gcode",
+        "platform": "FlashForge_DreamerNX.obj",
+        "platform_offset": [0, 0, 0],
+        "quality_definition": "flashforge_base",
+        "machine_extruder_trains":
+        {
+            "0": "ff_dreamernx_extruder_0"
+        }
```

Comment:
Reduntant, as it is already inherited by the `flashforge_base.def.json`.
```suggestion
```

Reply:
This is my first time. Thank you for pointing out the errors. I'll fix the mistakes tomorrow.
Thanks again!